src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -43,7 +43,12 @@ export const Autocomplete: AutocompleteType = ((props: AutocompleteProps) => {
           empty={errorProps.empty}
           suffix={
             <span className={style.suffix}>
-            {<ChevronDown className={style.arrowIcon}/>}
+            {<ChevronDown
```

Comment:
the icon color should be different when hovered/selected and when not

Reply:
I know, that in Zeplin it visually looks different, but actually it uses the same color states as text inside of input. And according to latest changes, text keeps it's color on hover state. Anyway I sent the question to Doron to be sure